src/llvm-coverage-instrumentation.md

```diff
@@ -223,9 +223,8 @@ details of the [_LLVM Coverage Mapping Format_][coverage-mapping-format]
 (Version 6).[^llvm-and-covmap-versions]
 
 [^llvm-and-covmap-versions]:
-The Rust compiler (as of <!-- date-check: --> Jan 2023) supports _LLVM Coverage Mapping Format_ 6.
-It was introduced in _LLVM 13_, which is, as of <!-- date-check: Jan 2023 --> this writing,
-the minimum supported LLVM version.
+The Rust compiler (as of <!-- date-check: --> Feb 2023) supports _LLVM Coverage Mapping Format_ 6.
+It was introduced in _LLVM 13_, which was the minimum supported LLVM version [until early 2023].
```

Comment:
I guess we don't have to mention the minimum supported version (and even the introduced version) anymore as the contributors/users now have to use the LLVM 14 or later anyway and the format version 6 is generally available there.